etcd/registry.go

```diff
@@ -16,13 +16,12 @@ package etcd
 
 import (
 	"context"
-	"encoding/json"
-	"sync"
-	"time"
-
+	"github.com/bytedance/sonic"
 	"github.com/cloudwego/hertz/pkg/app/server/registry"
 	"github.com/cloudwego/hertz/pkg/common/hlog"
 	clientv3 "go.etcd.io/etcd/client/v3"
+	"sync"
```

Comment:
Please format the code
> gofumpt -l -extra -w .

Reply:
Do I need format the code in local IDE then commit again ？

Reply:
yes, format the code locally by the command and push the commit .

Reply:
done it